Using gpu device 0: NVS 3100M
Job launching after 1.60 seconds in submission.
Running python job.
Changed into dir /home/lho/Htw/bachelor/thesis-code/spearmint/metric-test
Importing train.py
Running train.main()
WARNING (theano.sandbox.cuda.opt): Optimization Warning: Got the following error, but you can ignore it. This could cause less GpuElemwise fused together.
CudaNdarray_ptr_int_size: error when calling the gpu code. (invalid device function )
Traceback (most recent call last):
  File "/home/lho/Htw/bachelor/preparation/Spearmint/spearmint/launcher.py", line 244, in launch
    result = python_launcher(job)
  File "/home/lho/Htw/bachelor/preparation/Spearmint/spearmint/launcher.py", line 328, in python_launcher
    result = module.main(job['id'], params)
  File "/home/lho/Htw/bachelor/thesis-code/spearmint/metric-test/train.py", line 74, in main
    params['lmbda'][0])
  File "/home/lho/Htw/bachelor/thesis-code/spearmint/metric-test/train.py", line 61, in train
    metric_recorder=metric_recorder)
  File "/home/lho/Htw/bachelor/thesis-code/src/network.py", line 192, in SGD
    training_y[i*self.mini_batch_size: (i+1)*self.mini_batch_size]
  File "/usr/local/lib/python2.7/dist-packages/theano/compile/function.py", line 266, in function
    profile=profile)
  File "/usr/local/lib/python2.7/dist-packages/theano/compile/pfunc.py", line 511, in pfunc
    on_unused_input=on_unused_input)
  File "/usr/local/lib/python2.7/dist-packages/theano/compile/function_module.py", line 1465, in orig_function
    on_unused_input=on_unused_input).create(
  File "/usr/local/lib/python2.7/dist-packages/theano/compile/function_module.py", line 1160, in __init__
    optimizer_profile = optimizer(fgraph)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/opt.py", line 90, in __call__
    return self.optimize(fgraph)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/opt.py", line 81, in optimize
    ret = self.apply(fgraph, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/opt.py", line 195, in apply
    sub_prof = optimizer.optimize(fgraph)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/opt.py", line 81, in optimize
    ret = self.apply(fgraph, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/opt.py", line 1572, in apply
    q = deque(graph.io_toposort(fgraph.inputs, start_from))
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/graph.py", line 804, in io_toposort
    topo = general_toposort(outputs, deps)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/graph.py", line 745, in general_toposort
    reachable, clients = stack_search(deque(r_out), _deps, 'dfs', True)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/graph.py", line 521, in stack_search
    expand_l = expand(l)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/graph.py", line 731, in _deps
    d = deps(io)
  File "/usr/local/lib/python2.7/dist-packages/theano/gof/graph.py", line 793, in deps
    if obj not in iset:
KeyboardInterrupt
Problem executing the function
Job failed in 1.97 seconds.
Couldn't import dot_parser, loading of dot files will not be possible.
Anything printed here will end up in the output directory for job #2
{u'n_hidden_layer': array([140]), u'eta': array([ 0.055]), u'border': array([3]), u'lmbda': array([  5.00000000e-06])}
Job ID: 2
Training size: 22
Validation size: 5
(<type 'exceptions.KeyboardInterrupt'>, KeyboardInterrupt(), <traceback object at 0x7f714c0b8cb0>)
